.github/workflows/ci.yml

```diff
@@ -70,3 +70,29 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: bnjbvr/cargo-machete@main
+
+  bench-riscv:
```

Comment:
Benches for every small commit is too much, for example we don't run nova benchmarks as a workflow

Reply:
I run this part in my local and it's fast enough to run on CI. It takes 5 minutes CI, I think it's alright. 
Because I want to have continuous benchmarking flow on CI. 
I can add a label feature, whenever the benchmark label is added, the CI workflow benchmark will be executed.

Reply:
added in https://github.com/nexus-xyz/nexus-zkvm/pull/265

Reply:
added in https://github.com/nexus-xyz/nexus-zkvm/pull/265